furios_gallery/gallery_window.py

```diff
@@ -123,6 +124,25 @@ def __init__(self, *args, **kwargs):
         self.navigation_view.connect('pushed', self.on_navigation_changed)
 
         self.present()
+    
+    def on_page_popped(self, navigation_view, page):
+        # 1) Cancel any timeouts/idle callbacks the page created
```

Comment:
Nit: get rid of the 1) 2) 3) prefixes
If the order changes then the comment will need changes, showing up as a useless addition in history